membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -81,6 +80,39 @@ object AttributesFromZuora extends LoggingWithLogstashFields {
     }
   }
 
+
+
+  def dynamoUpdateRequired(dynamoAttributes: Option[Attributes], zuoraAttributes: Option[Attributes], identityId: String, projectedExpiry: => DateTime = DateTime.now().plusDays(14)) = {
```

Comment:
notsure about projectedExpiry as it doesn't explain what it is, maybe idealExpiry or latestExpiryLimit? not 100% sure